src/osx/cocoa/overlay.mm

```diff
@@ -78,6 +78,10 @@ - (instancetype)initWithContentRect:(NSRect)contentRect styleMask:(NSUInteger)st
     return self;
 }
 
+- (void)dealloc
+{
+   [super dealloc];
+}
```

Comment:
Just for my own education, could you please explain why is this necessary? I.e. wouldn't superclass dealloc be used by default anyhow?

Reply:
I got a warning without that, that my property would not be properly released, apparently the code for this only gets generated if there's a destructor. I couldn't find any notice about this behavior anywhere, but I followed clang's suggestion here ..